distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -852,11 +899,11 @@ public void testCswDurableSubscription() throws Exception {
         Bundle bundle = bundleService.getBundle("spatial-csw-endpoint");
         bundle.stop();
         while (bundle.getState() != Bundle.RESOLVED) {
-            Thread.sleep(1000);
+            sleep(1000);
         }
         bundle.start();
         while (bundle.getState() != Bundle.ACTIVE) {
-            Thread.sleep(1000);
+            sleep(1000);
```

Comment:
This line is a just a reformat. This test doesn't use the stub server or a ChunkedContent object that contains the Interrupted catch.